cosmos/scripts/converting_log_to_csv.py

```diff
@@ -78,22 +78,22 @@ def parse_line_for_formatting(line, writer):
         elif 'Count=' in metrics:
             count = metrics.replace('Count=', '')
         elif 'Max=' in metrics:
-            max_in_micro_sec = metrics.replace('Max=', '')
+            max_in_milli_sec = format(float(metrics.replace('Max=', '')) / 1000, '.4f')
         elif 'Min=' in metrics:
-            min_in_micro_sec = metrics.replace('Min=', '')
+            min_in_milli_sec = format(float(metrics.replace('Min=', '')) / 1000, '.4f')
         elif 'Avg=' in metrics:
-            avg_in_micro_sec = metrics.replace('Avg=', '')
+            avg_in_milli_sec = format(float(metrics.replace('Avg=', '')) / 1000, '.4f')
         elif '99.99=' in metrics:
-            p9999_in_micro_sec = metrics.replace('99.99=', '')
+            p9999_in_milli_sec = format(float(metrics.replace('99.99=', '')) / 1000, '.4f')
         elif '99.9=' in metrics:
-            p999_in_micro_sec = metrics.replace('99.9=', '')
+            p999_in_milli_sec = format(float(metrics.replace('99.9=', '')) / 1000, '.4f')
         elif '99=' in metrics:
-            p99_in_micro_sec = metrics.replace('99=', '')
+            p99_in_milli_sec = format(float(metrics.replace('99=', '')) / 1000, '.4f')
         elif '90=' in metrics:
-            p90_in_micro_sec = metrics.replace('90=', '')
+            p90_in_milli_sec = format(float(metrics.replace('90=', '')) / 1000, '.4f')
 
-    row_in_csv = [date, time, operation, rps, count, max_in_micro_sec, min_in_micro_sec, avg_in_micro_sec,
-                  p90_in_micro_sec, p99_in_micro_sec, p999_in_micro_sec, p9999_in_micro_sec]
+    row_in_csv = [date, time, operation, rps, count, max_in_milli_sec, min_in_milli_sec, avg_in_milli_sec,
+                  p90_in_milli_sec, p99_in_milli_sec, p999_in_milli_sec, p9999_in_milli_sec]

```

Comment:
The milliseconds are getting rounded down in the 

for example:
This is the YCSB output:
[READ], AverageLatency(us), 2326.295872
[READ], MinLatency(us), 749
[READ], MaxLatency(us), 295174143
[READ], 95thPercentileLatency(us), 1608
[READ], 99thPercentileLatency(us), 44671

This is what is showing in the aggregation.csv
Operation,Count,Throughput,Min(millisecond),Max(millisecond),Avg(millisecond),P95(millisecond),P99(millisecond)
READ,750000,428,0.749,295174.143,2,1,44